lib/intercom/request.rb

```diff
@@ -75,8 +75,12 @@ def execute(target_base_url=nil, username:, secret: nil)
           rescue Intercom::RateLimitExceeded => e
             if @handle_rate_limit
               seconds_to_retry = (@rate_limit_details[:reset_at] - Time.now.utc).ceil
-              sleep seconds_to_retry unless seconds_to_retry < 0
-              retry unless (retries -=1).zero?
+              if (retries-=1) < 0
```

Comment:
A bit of personal preference but I think it's more readable if we write this as `if (retries -= 1) < 0`. The whitespace around the operators make it easier for me.